stories/basic.stories.tsx

```diff
@@ -50,6 +72,49 @@ const BasicForm = () => {
           ]}
         />
       </Form.Item>
+      <Form.List name="children" rules={[rule as unknown as ValidatorRule]}>
+        {(childrenFields, { add }, { errors }) => {
+          return (
+            <div>
```

Comment:
Can you use `antd` `Container` here and add some left padding so we get indentation., a sense of nested object.